### ISSUE TITLE
Unicode issue on highlighting search text

### ISSUE THREAD

**Issue by pincr** [openscad/openscad#2795]
Minor problem when highlighting the search text. Every national char (more bytes in UTF-8 encoding?) makes shift on  highlighting the search text:

![snimek z 2019-02-17 09-56-32](https://user-images.githubusercontent.com/6544350/52910628-a1a7ec80-329a-11e9-9426-2a436377ed85.png)

It is issue only on highlighting. Previous, next and replacement works fine.

Test example:

```openscad
module test() {
	sphere(10);
}

// Text with ony ASCII characters
translate([10,-10,0]) test();

// Text with national char: test "Č" test
translate([0,10,-10]) test();

// Another text with national char: test "Ř" test
translate([-10,0,10]) test();

text("Jet another text with national char: test 'Ů' test");

translate([-10,-10,-10])
test();
```
OpenSCAD version 2019.02.16.nightly (git 38f3279)
Fedora 28 (64-bit only) - https://download.opensuse.org/repositories/home:/t-paul/Fedora_28/

**Comment by MisterAwesome23** [comment]
I would like to fix this GUI bug. Can someone help me get started with the codebase. I'm a new member :)

**Comment by kintel** [comment]
A great start would be to build OpenSCAD from source and reproduce this using your self-built binary. The join the IRC channel to get advice going from there.

**Comment by MisterAwesome23** [comment]
Okay. I'll set up self-built binary and reproduce the bug. I'll be updated on IRC as soon as I set it all up. Thanks

**Comment by t-paul** [comment]
Fixed via #3071.
